src/NuGetGallery.Core/Infrastructure/TableErrorLog.cs

```diff
@@ -14,206 +14,220 @@
-                long pos = _entityList.Add(entity);
-                return pos.ToString(CultureInfo.InvariantCulture);
+                error.Form.Clear();
             }
 
-            private void Obfuscate(Error error)
+            //ServerVariables overrides requiring context from the http request should be handled in NuGetGallery.QuietLog
+            var elmahException = error.Exception as ElmahException;
+            if (elmahException != null)
             {
-                error.User = string.Empty;
-                if (error.Form != null)
+                var piiServerVaribles = elmahException.ServerVariables;
+                foreach (var key in piiServerVaribles.Keys)
                 {
-                    error.Form.Clear();
+                    error.ServerVariables[key] = piiServerVaribles[key];
                 }
+            }
 
-                //ServerVariables overrides requiring context from the http request should be handled in NuGetGallery.QuietLog
-                var elmahException = error.Exception as ElmahException;
-                if (elmahException != null)
-                {
-                    var piiServerVaribles = elmahException.ServerVariables;
-                    foreach (var key in piiServerVaribles.Keys)
-                    {
-                        error.ServerVariables[key] = piiServerVaribles[key];
-                    }
-                }
+            error.ServerVariables["ALL_HTTP"] = string.Empty;
+            error.ServerVariables["ALL_RAW"] = string.Empty;
+
+            error.ServerVariables["AUTH_USER"] = string.Empty;
+            error.ServerVariables["LOGON_USER"] = string.Empty;
+            error.ServerVariables["REMOTE_USER"] = string.Empty;
+
+            error.ServerVariables["REMOTE_ADDR"] = Obfuscator.ObfuscateIp(error.ServerVariables["REMOTE_ADDR"]);
+            error.ServerVariables["REMOTE_HOST"] = Obfuscator.ObfuscateIp(error.ServerVariables["REMOTE_HOST"]);
+            error.ServerVariables["LOCAL_ADDR"] = Obfuscator.ObfuscateIp(error.ServerVariables["LOCAL_ADDR"]);
 
-                error.ServerVariables["AUTH_USER"] = string.Empty;
-                error.ServerVariables["LOGON_USER"] = string.Empty;
-                error.ServerVariables["REMOTE_USER"] = string.Empty;
+            error.ServerVariables["HTTP_X_NUGET_APIKEY"] = string.Empty;
 
-                error.ServerVariables["REMOTE_ADDR"] = Obfuscator.ObfuscateIp(error.ServerVariables["REMOTE_ADDR"]);
-                error.ServerVariables["REMOTE_HOST"] = Obfuscator.ObfuscateIp(error.ServerVariables["REMOTE_HOST"]);
-                error.ServerVariables["LOCAL_ADDR"] = Obfuscator.ObfuscateIp(error.ServerVariables["LOCAL_ADDR"]);
+            var forwardedIps = error.ServerVariables["HTTP_X_FORWARDED_FOR"].Split(',');
+            var obfuscatedIps = new List<string>();
+            foreach(var ip in forwardedIps)
+            {
+                obfuscatedIps.Add(Obfuscator.ObfuscateIp(ip));
             }
```

Comment:
If you'd like, this can be nicely LINQified:

```suggestion
            var obfuscatedIps = forwardedIps.Select(Obfuscator.ObfuscateIp);
```

Just a thought. If you choose not to go with this approach, please add a space between `foreach` and `(`.